### ISSUE TITLE
One click MAS upgrade from 8.9.3 to 8.10 has failed with error has no attribute '8.10.x'

### ISSUE THREAD

**Issue by ekambaraml** [ibm-mas/ansible-devops#741]
command used:
docker run -ti --rm --pull always quay.io/ibmmas/cli
export MAS_CHANNEL=8.10.x
ansible-playbook ibm.mas_devops.oneclick_upgrade

With following Error:

```
TASK [ibm.mas_devops.suite_upgrade : iot : Check that upgrade is supported from current channel] **********************************************************************************************************************************************************************
Wednesday 29 March 2023  12:31:51 +0000 (0:00:00.115)       0:00:15.428 ******* 
fatal: [localhost]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'dict object' has no attribute '8.10.x'\n\nThe error appears to be in '/opt/app-root/lib64/python3.9/site-packages/ansible_collections/ibm/mas_devops/roles/suite_upgrade/tasks/check_app_compatibility.yml': line 20, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: \"{{ check_app.id }} : Check that upgrade is supported from current channel\"\n  ^ here\nWe could be wrong, but this one looks like it might be an issue with\nmissing quotes. Always quote template expression brackets when they\nstart a value. For instance:\n\n    with_items:\n      - {{ foo }}\n\nShould be written as:\n\n    with_items:\n      - \"{{ foo }}\"\n"}

NO MORE HOSTS LEFT ****************************************************************************************************************************************************************************************************************************************************

PLAY RECAP ************************************************************************************************************************************************************************************************************************************************************
localhost                  : ok=33   changed=0    unreachable=0    failed=1    skipped=25   rescued=0    ignored=0 
```

**Comment by ekambaraml** [comment]
Fix:

Run the following command to force refresh the playbooks.

ansible-galaxy collection install ibm.mas_devops --force

Then run the upgrade commmands
```
export MAS_CHANNEL=8.10.x
ansible-playbook ibm.mas_devops.oneclick_upgrade
```

Thanks to @whitfiea for helping to fix.

**Comment by whitfiea** [comment]
Fixed in ansbiel-devops [12.16.1](https://github.com/ibm-mas/ansible-devops/releases/tag/12.16.1) and cli [3.18.1](https://github.com/ibm-mas/cli/releases/tag/3.18.1)
